bodhi/client/__init__.py

```diff
@@ -295,7 +327,7 @@ def new(user, password, url, **kwargs):
                        True.
         kwargs (dict): Other keyword arguments passed to us by click.
     """
-
+    kwargs.pop('debug')
```

Comment:
Is this what you meant by being a larger change? If so, a cleaner way to do this is to add ```debug``` as a parameter to each of these functions so it doesn't end up in ```kwargs```.

Reply:
Good idea, will fix